### DISCUSSION TITLE
Update awscli formula to use source tarballs

### DISCUSSION THREAD

**Discussion by imgx64** [Homebrew/discussions#2061] [Everyday usage]
The [current awscli formula][1] gets the source from specific git tags since [v2 is not published to PyPI][2].

Luckily, The AWS CLI team has posted a [proposal for releasing source tarballs for v2][3] that can be built and installed using `./configure && make && make install`. This will make packaging it much easier.

Can the maintainer of the awscli formula (@chenrui333 ?) look into using this instead of the current method, and post any comments or suggestions to the proposal?

@cho-m I'm tagging you because I see you've done a [major restructuring of the formula recently][4], so you might be interested.

[1]: https://github.com/Homebrew/homebrew-core/blob/e8e295573bb20787d889a6664c3859d7dba6aa97/Formula/awscli.rb
[2]: https://github.com/aws/aws-cli/issues/4947
[3]: https://github.com/aws/aws-cli/pull/6352
[4]: https://github.com/Homebrew/homebrew-core/pull/83109

**Comment by carlocab** [comment]
Specific formulae don't have assigned maintainers. If you have specific changes you'd like to propose, please open a pull request.
